hack/generator/pkg/astmodel/armconversion/convert_from_arm_function_builder.go

```diff
@@ -224,6 +225,99 @@ func (builder *convertFromARMBuilder) ownerPropertyHandler(
 	return []dst.Stmt{result}
 }
 
+func (builder *convertFromARMBuilder) flattenedPropertyHandler(
```

Comment:
Some of the other methods don't do this (shame on me), but it might be nice to have a skeleton of what code is actually generated from this in the comment of the method.

Reply:
It's hard to do that here because mostly it relies on the other conversion handlers to generate the inner conversion. I'll do a sketch 🙂

Reply:
Yeah it's tricky because the guts are actually not generated by this, but yeah just a really simple skeleton that sketches out what "flattening a property" actually entails at a high level.

Reply:
Added a very high-level description.